theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -285,22 +286,24 @@ def test_pooling():
                     a = f1(data).__array__()
 
                     b = f2(data).__array__()
-                    utt.assert_allclose(a, b)
+                    assert numpy.allclose(a, b,
+                                          atol=numpy.finfo(numpy.float32).eps)
 
         # Test the grad
         for shp in [(1, 1, 2, 2),
                     (1, 1, 3, 3)]:
             data = numpy.random.normal(0, 1, shp).astype("float32") * 10
 
-            ws = 2
-            stride = 2
-            if pad[0] > stride or pad[1] > stride:
+            ws = theano.shared(numpy.array([2, 2]))
```

Comment:
I want to test if the op accepts tensor variables. @lamblin suggested to use shared variables when you pass it to verify grad.